sci2zero/TeslaRIS-frontend
# Feature/proceedings publications load

## Description

Added wizard support for proceedings publications.

## How should this be tested?

Navigate to loading wizard. If proceedings publication is loaded you will be able to finish loading process and see it on the scientific results page.

## Checklist
